### DISCUSSION TITLE
Roku app not working properly with 10.8.1 release

### DISCUSSION THREAD

**Discussion by jdohare** [jellyfin/jellyfin#8034] [General]
Not sure if this is the proper place to report, but Roku app (1.4 build 12 on Roku Ultra) has problems playing back HEVC x265 encoded video using server 10.8.1.   Get "Error During Playback" and flickering background, the latter because it seems to be iterating through all episodes in a season trying (unsuccessfully) to find one that can play without error.  Reverting back to 10.8.0 resolves the issue.

Looking at the logs, it appears as if the file plays under 10.8.1 as FFmpeg.Transcode and under 10.8.0 as FFmpeg.DirectStream

**Comment by goldbondxyz** [comment]
OK I thought it was only me and possibly broken transcoding.  Thanks for raising.
